src/async_impl/client.rs

```diff
@@ -129,20 +213,34 @@ impl ClientBuilder {
         })
     }
 
+    /// Use native TLS backend.
+    #[cfg(feature = "default-tls")]
+    pub fn use_default_tls(mut self, tls: Option<TlsConnectorBuilder>) -> ClientBuilder {
```

Comment:
I'd remove the argument (and on `use_rustls`), to be conservative. That way, if they have breaking changes, we don't need to break as well just to upgrade.